client/app/vps/vps.js

```diff
@@ -131,6 +131,35 @@ angular.module('managerApp').config(($stateProvider) => {
         },
       },
     })
+    .state('iaas.vps.detail.cloud-database', {
+      url: '/cloud-database',
+      redirectTo: 'iaas.vps.detail.cloud-database.list',
+      views: {
+        vpsContent: {
+          template: '<div ui-view="vpsCloudDatabaseContent"></div>'
+        }
+      }
+    })
+    .state('iaas.vps.detail.cloud-database.list', {
+      url: '/',
+      views: {
+        vpsCloudDatabaseContent: {
+          templateUrl: 'app/vps/cloud-database/vps-cloud-database.html',
+          controller: 'VpsCloudDatabaseCtrl',
```

Comment:
Could have use a component instead

Reply:
I don't know what a "component" is. Could you redirect me to a doc ?

Reply:
https://github.com/ovh-ux/ovh-ui-angular/tree/master/packages/oui-button/src for example

You can add a route to a Component this way: https://github.com/ovh-ux/ovh-manager-dedicated/blob/3c049b4bdd628f866fb30b58c68bb9516c04bf11/client/app/account/billing/billingApp.module.js#L122

Reply:
:+1: